datasources/signals.datasource

```diff
@@ -20,7 +10,8 @@ SCHEMA >
     `value` Float32 `json:$.value`,
     `unit` String `json:$.unit`,
     `job_id` String `json:$.job_id`,
-    `ingested_at` DateTime `json:$.ingested_at`
+    `ingested_at` DateTime `json:$.ingested_at`,
+    `signal` String `json:$.signal` DEFAULT ''
```

Comment:
I guess DEFAULT is not supported for NDJSON columns (could you double check?) due to not being implemented in the API.

I guess we could use `Nullable(String)` instead.

Reply:
You're correct, the records go to quarantine. I intentionally did it this way to avoid the performance challenges associated with using Nullable columns in ClickHouse. But it is not supported when using NDJSON so I'm changing it.

Reply:
Changed!